### PR TITLE
Polish docs.

### PR DESCRIPTION
**Pull request by izeye**

### CODE REVIEW

**Comment by wilkinsona** [comment]
Thanks very much. Merged with a few amendments.

**Comment by izeye** [comment]
@wilkinsona Thanks for the nice tool!

File: docs/src/docs/asciidoc/contributing.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [[contributing]]
 == Contributing
 
-Spring REST Docs is intended to make is easy for you to produce high-quality documentation
+Spring REST Docs is intended to make easy for you to produce high-quality documentation
```
Comment (otrosien) [inline docs/src/docs/asciidoc/contributing.adoc]: to make it easy :)

Comment (wilkinsona) [inline docs/src/docs/asciidoc/contributing.adoc]: Thanks. I caught that one as part of the merge. See: https://github.com/spring-projects/spring-restdocs/commit/6f4fe4ea644d77d66985c249102c8abd3d44c59d#diff-1c885f4a0e9ffe753bc7f61ba4d69f3bR4

**Pull request closed without merging**